tests/integration_tests/suites.yaml

```diff
@@ -0,0 +1,62 @@
+security:
+  - manager_tests/test_secured_broker.py
+  - manager_tests/test_secured_rest.py
+  - manager_tests/test_secured_rest_authentication.py
+  - manager_tests/test_secured_rest_authorization.py
+  - manager_tests/test_secured_rest_base.py
+  - manager_tests/test_secured_rest_cert_location.py
+  - manager_tests/test_secured_rest_no_userstore.py
+  - manager_tests/test_secured_rest_user_certificate.py
+
+orchestration:
+  - agent_tests/test_existing_vm.py
+  - agent_tests/test_policies_with_diamond.py
+  - agentless_tests/test_agent_alive_verification.py
+  - agentless_tests/test_n_instances.py
+  - agentless_tests/test_policies.py
+  - agentless_tests/test_scale_workflow.py
+  - agentless_tests/test_wf_api.py
+  - agentless_tests/test_workflow.py
+  - agentless_tests/test_subgraph_retries.py
+  - agentless_tests/test_task_retries.py
+  - agentless_tests/test_events.py
+  - agentless_tests/test_execution_logging.py
+  - agentless_tests/test_executions.py
+
+backwards:
+  - agentless_tests/test_backwards_agent_installation_blueprints.py
+  - agentless_tests/test_backwards_rest_api.py
+
+deployments:
+  - agentless_tests/test_deployment_logs.py
+  - agentless_tests/test_deployment_nodes.py
+  - agentless_tests/test_deployment_workflows.py
+  - agentless_tests/test_uninstall_deployment.py
+
+rest:
+  - agentless_tests/test_rest_api.py
+  - agentless_tests/test_rest_service_list_filters.py
+  - agentless_tests/test_rest_service_pagination.py
+  - agentless_tests/test_rest_service_sort.py
+  - manager_tests/test_rest_plugins.py
+
+manager:
+ - agent_tests/test_manager_maintenance_mode.py
+ - agent_tests/test_plugin_workdir.py
+ - agentless_tests/test_context_properties.py
+ - agentless_tests/test_download_blueprint.py
+ - agentless_tests/test_elasticsearch.py
+ - agentless_tests/test_get_instance_ip.py
+ - agentless_tests/test_operation_mapping.py
+ - agentless_tests/test_plugins.py
```

Comment:
consider plugins suite even though not using actual plugins but only checking plugins infra